http-add-on/values.yaml

```diff
@@ -163,6 +163,17 @@ interceptor:
   scaledObject:
     # -- The interval (in milliseconds) that KEDA should poll the external scaler to fetch scaling metrics about the interceptor
     pollingInterval: 1
+    pendingRequestsTrigger:
```

Comment:
Nit, would be nice to add a short comment with and explanation, the similar that's used for cpu/mem, with a note that users mist likely don't want to use this trigger